update-bitwarden.sh

```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
+ask()
+{
+  declare -g $1="$2"
+  if [ -z "${!1}" ]; then
+    echo "$3"
+    read $1
+  fi
+}
```

Comment:
To stay compatible with bash 3.x I would suggest using something like:

```suggestion
ask () {
  local __resultVar=$1
  local __result="$2"
  if [ -z "$2" ]; then
    read -p "$3" __result
  fi
  eval $__resultVar="'$__result'"
}
```

Reply:
I included it in my commit. Thanks for the hint